Common/Symbol.cs

```diff
@@ -30,6 +31,8 @@ namespace QuantConnect
     [ProtoContract(SkipConstructor = true)]
     public sealed class Symbol : IEquatable<Symbol>, IComparable
     {
+        private static readonly SecurityDefinitionSymbolResolver _securityDefinitionSymbolResolver = new();
```

Comment:
Given the algorithm also has it's own instance, think we can follow the pattern set by the MarketHDB & SPDB being a singleton